src/MODULE.bazel

```diff
@@ -1 +1,2 @@
 bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "rules_pkg", version = "1.0.1")
```

Comment:
You should delete this file. We probably aren't going to move to MODULE.bazel anytime soon. 

I remember reading an issues about `rules_python` and `pybind11_protobuf` having issues with locally installed toolchain somewhere when using MODULE.